### PR TITLE
discard outputs coming from onbuild command on buildah-from --quiet

### LINKED ISSUE #2014: buildah-from with ONBUILD and --quiet: is not quiet
This seems to be an undefined corner case:
```console
# cat Dockerfile
FROM alpine
ONBUILD RUN date
# buildah bud --format=docker --quiet -t foo .
baa08aa3383dc647dd543aed0c5d2e8bae1d7a63b060e869dbc68524ea862ee1
# buildah from --quiet foo
STEP 1: RUN date                 <<< unexpected
Tue Dec 10 21:56:15 UTC 2019     <<< unexpected
foo-working-container            <<< expected
```
Note that there are two different unexpected lines in the output: the `STEP` one, and the actual output of the command itself.

`buildah-from(1)` describes `--quiet` as follows:
```
   --quiet, -q

   If an image needs to be pulled from  the  registry,  suppress  progress output.
```
...so technically speaking the `ONBUILD` behavior is undefined; but I think one can have a reasonable expectation that `--quiet` from `bud` will carry over onto `from`. Plus, docker (19.0.3 - I can't get a RHEL7 virt to test 13.c) seems to respect `--quiet`:
```console
# docker build --quiet .
sha256:7634f230b6506afeb29e8dbfce059fecfdb7f71ce50ff19dcf43f676ad81ce79
```

### PR DESCRIPTION
**Pull request by boaz0**
closes #2014

### CODE REVIEW

File: tests/bud.bats [hunk 1]
```diff
@@ -860,7 +860,7 @@ load helpers
   run_buildah inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
   expect_output '["RUN touch /onbuild1" "RUN touch /onbuild2"]'
   run_buildah from --quiet ${target}
-  cid=${lines[2]}              # FIXME FIXME FIXME: see issue #2014
+  cid=${lines[0]}              # FIXME FIXME FIXME: see issue #2014
```
Comment (rhatdan) [inline tests/bud.bats]: Why not remove the comment here?

**Comment by rhatdan** [comment]
LGTM once the test comment is fixed.

File: tests/bud.bats (hunk 1 above)
Comment (boaz0) [reply tests/bud.bats]: Oops missed that part :rofl:

**Comment by rhatdan** [comment]
@nalind @QiWang19 @TomSweeneyRedHat @vrothberg PTAL

**Comment by QiWang19** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
Thanks @boaz0

**Pull request closed without merging**